core/app.js

```diff
@@ -332,6 +332,32 @@ app.use(_.get("/employees",function *(){
   }
 }));
 
+app.use(_.get("/rss",function *(){
+  var rss = require('rss');
+  var newsbox = yield require('./app_modules/newsbox').list;
+  var feed = new rss({
+    title: 'NewsBox',
+    feed_url: 'http://' + ENV.HLRDESK_HOST + ENV.PORT + '/rss',
```

Comment:
This URL will be incorrect on the production site, since the port will be an internal-only port, and it's running on HTTPS.

Use app_modules/utils.js for an example of generating an appropriate URL. For example:

`utils.gen_url(ENV.HLRDESK_HOST, ENV.PORT, '/rss', !ENV.HLRDESK_DEV)`